### PR TITLE
book

### PR DESCRIPTION
**Pull request by quannb99**
## Related Tickets:
- [#29308](https://edu-redmine.sun-asterisk.vn/issues/29308)

## WHAT this PR does?
- Display random 3 books in home page

## Notes Impacted Areas:
- N/A.

## Attachments:
- random

### CODE REVIEW

**Comment by quannb99** [comment]
ready

File: resources/views/home.blade.php [hunk 1]
```diff
@@ -1 +1,47 @@
 @extends('layouts.app')
+
+@section('content')
+    <section class="padding-bottom-18">
+        <div class="container">
+            <div class="row">
+                @foreach ($books as $book)
+                    <div class="col-lg-4 col-md-6 col-sm-12">
+                        <div class="product-box-layout1">
+                            <figure class="item-figure"><a href="#"><img src="{{ $book->img_path }}" alt="Product"></a>
```
Comment (thaonp104) [inline resources/views/home.blade.php]: i18n 
```suggestion
                            <figure class="item-figure"><a href="#"><img src="{{ assert($book->img_path) }}" alt="Product"></a>
```

File: routes/web.php [hunk 2]
```diff
@@ -19,6 +19,8 @@
     Route::group(['prefix' => 'admin', 'middleware' => 'admin'], function () {
 
         Route::get('/', 'HomeController@admin_index')->name('admin_index');
+
+        Route::resource('book', 'BookController');
```
Comment (huyenttm-0901) [inline routes/web.php]:
```suggestion
        Route::resource('books', 'BookController');
```

**Review by huyenttm-0901 — COMMENTED** [review]
update status redmine

**Comment by quannb99** [comment]
ready

**Pull request merged**